src/main/scala/tech/cryptonomic/conseil/tezos/ApiOperations.scala

```diff
@@ -282,5 +282,24 @@ object ApiOperations {
           .headOption
     )
 
+  /**Counts accounts in the db**/
+  def countAccounts: Future[Int] =
+    dbHandle.run(TezosDb.countAcounts)
+
+  /**Counts accounts in the db**/
```

Comment:
plesae, update the docs and use the correct syntax (no double `**/` at the end)

Reply:
removed method, updated comment in the new one